pkg/config/threescale.go

```diff
@@ -128,4 +116,19 @@ func (t *ThreeScale) GetReplicasConfig(inst *integreatlyv1alpha1.RHMI) map[strin
 		"zyncApp":         3,
 		"zyncQue":         3,
 	}
+
+	if resources.RunningInProw(inst) {
+		setDefaultNumberOfReplicas(1, threeScaleComponents)
+	}
+	if inst.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManaged) {
```

Comment:
```suggestion
	if inst.Spec.Type != string(integreatlyv1alpha1.InstallationTypeManagedApi) {
```

I'm not sure if this is used in practice, but if the installation type is `Workshop`,  it would keep the number of replicas for RHOAM. Might be safer to check that the installation type is *not* Managed API